app/src/lang/translations/en-US.yaml

```
@@ -1631,6 +1642,14 @@ select_field_type: Select a field type
 select_interface: Select an interface
 select_display: Select a display
 settings: Settings
+register: Register
+registration_successful_headline: Success!
+registration_successful_note: You may now proceed to the sign in page.
+registration_successful_check_email_note: |
+  Please note that before you can sign in, you'll need to verify your email address by clicking on the verification link sent to you.
+dont_have_an_account: Don't have an account?
+already_have_an_account: Already have an account?
```

Translation keys should be somewhat neutral (and existing ones generally are)
```suggestion
no_account: Don't have an account?
already_an_account: Already have an account?
```